RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -163,17 +163,94 @@ class RNUnifiedContacts: NSObject {
 
       callback([error.localizedDescription, NSNull()])
     }
-
   }
-
-
-
-
-
+  
+  @objc func createContact(contactData: NSDictionary, callback: (NSObject) -> () ) -> Void {
+    
+    let mutableContact = CNMutableContact()
+    let contactStore = CNContactStore()
+    let req = CNSaveRequest()
+    
+    // TODO:
+    // Extend method to handle more fields
+    //
+    //
+    mutableContact.givenName = contactData["givenName"] as! String
+    mutableContact.familyName = contactData["familyName"] as! String
+    mutableContact.organizationName = contactData["organizationName"] as! String
+    
+    for number in contactData["phoneNumbers"] as! NSArray {
+      let transformedNumber = transformNumber( number as! NSDictionary )
+      mutableContact.phoneNumbers.append( transformedNumber )
+    }
+    
+    for email in contactData["emailAddresses"] as! NSArray {
+      let transformedEmail = transformEmail ( email as! NSDictionary )
+      mutableContact.emailAddresses.append( transformedEmail )
+    }
+    
+    do {
+      req.addContact(mutableContact, toContainerWithIdentifier:nil)
+      try contactStore.executeSaveRequest(req)
+      print("Successfully created contact")
+      callback( [NSNull(), ""] )
```

Comment:
problem with `getContact` is that it returns a Dictionary based on `CNContact` object. I don't have the `CNContact` here, only `CNMutableContact`. So we need to change the Dictionary transformation some. 

Also it would be a good idea to implement the `CNContactStoreDidChangeNotification` here